content/3_Silver/Two_Pointers.mdx

```diff
@@ -280,3 +280,103 @@ public class Books {
 ## Problems
 
 <Problems problems="ad" />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    What is the two pointers algorithm?
+    <Quiz.Answer>
+      Keeping track of two indices to find a subarray that satisfies some condition.
+      <Quiz.Explanation>
+        Incorrect. The indices don't necessarily have to represent a subarray, they could just represent elements in the array.
+	      
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Keeping track of two indices to find a pair of indices that satisfy some condition.
+      <Quiz.Explanation>
+        Correct.
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A way to answer range sum queries in $\mathcal{O}(1)$ time.
+      <Quiz.Explanation>
+        Incorrect. You might be thinking of [Prefix Sums](/silver/prefix-sums), another silver topic.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A divide and conquer algorithm that splits an array into two halves to efficiently find an element satisfying a certain condition.
+      <Quiz.Explanation>
+        Incorrect.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What should go in place of `???`?
+
+	```
+	/*
+	 * find two elements at distinct positions in an array 
+	 * which add up to a target sum
+	 */
+	  
+	nums.sort()
+
+	l = 0
+	r = n - 1
+
+	while (???) {
+		if (nums[l] + nums[r] == target) {
+			found pair
+			break
+		}
+		else if (nums[l] + nums[r] > target) {
+			r--
+		} else {
+			l++
+		}
+	}
```

Comment:
```suggestion
	/*
	 * find two elements at distinct positions in an array 
	 * which add up to a target sum
	 */
	  
	nums.sort()

	l = 0
	r = n - 1

	while (???) {
		if (nums[l] + nums[r] == target) {
			found pair
			break
		}
		else if (nums[l] + nums[r] > target) {
			r--
		} else {
			l++
		}
	}
```
make this a code block with ticks

Reply:
aren't there already ticks?
<img width="296" alt="image" src="https://user-images.githubusercontent.com/81596991/143988306-56b30805-700a-4f3d-9e11-c354700c3005.png">

Reply:
oop would be nicer for language formatting instead of plaintext

Reply:
added for py and java